### PR TITLE
Original data return from write_* functions no longer has UTC-adjusted date/times

### PR DESCRIPTION
**Pull request by jmobrien**
Addresses secondary issue mentioned in #702.  

Noting ahead of time: it's possible changes made here might not be necessary/a good fit if other changes are made in response to #702, and if so this PR can be ignored.

Relatedly, if other #702-related changes (e.g., reverting to older behavior, and/or adding a new selector argument for how conversion is done) are both desired and easier if just added to this PR, I'd be glad to do that.

### CODE REVIEW

File: R/haven-sas.R [hunk 1]
```diff
@@ -146,8 +149,6 @@ write_xpt <- function(data, path, version = 8, name = NULL, label = attr(data, "
 
 validate_sas <- function(data) {
   stopifnot(is.data.frame(data))
-
-  adjust_tz(data)
```
Comment (gorcha) [inline R/haven-sas.R:150]: Now that it's not adjusting the timezone I'd prefer that it just returns `invisible(data)` for so there's a consistent return value for all of the `validate_*()` functions

File: R/haven-spss.R [hunk 2]
```diff
@@ -165,8 +166,6 @@ validate_sav <- function(data, call = caller_env()) {
       call = call
     )
   }
-
-  adjust_tz(data)
```
Comment (gorcha) [inline R/haven-spss.R:169]: Now that it's not adjusting the timezone I'd prefer that it just returns `invisible(data)` for so there's a consistent return value for all of the `validate_*()` functions

File: R/haven-stata.R [hunk 3]
```diff
@@ -147,7 +151,6 @@ validate_dta <- function(data, version, call = caller_env()) {
     )
   }
 
-  adjust_tz(data)
```
Comment (gorcha) [inline R/haven-stata.R:150]: Now that it's not adjusting the timezone I'd prefer that it just returns `invisible(data)` for so there's a consistent return value for all of the `validate_*()` functions

**Review by gorcha — COMMENTED** [review]
Hi @jmobrien, thanks for the PR! All looking good, just one thing - I'd prefer all the `validate_*()` functions to return `invisible(data)` so they have a consistent return value.

File: R/haven-sas.R (hunk 1 above)
Comment (jmobrien) [reply R/haven-sas.R:150]: Done, in [41f21fe](https://github.com/tidyverse/haven/pull/703/commits/41f21fe0eb5bea54b02206610dcdffedf2ae14b7)

File: R/haven-spss.R (hunk 2 above)
Comment (jmobrien) [reply R/haven-spss.R:169]: Done, in [41f21fe](https://github.com/tidyverse/haven/pull/703/commits/41f21fe0eb5bea54b02206610dcdffedf2ae14b7)

File: R/haven-stata.R (hunk 3 above)
Comment (jmobrien) [reply R/haven-stata.R:150]: Done, in [41f21fe](https://github.com/tidyverse/haven/pull/703/commits/41f21fe0eb5bea54b02206610dcdffedf2ae14b7)

**Pull request merged**